src/storage/src/hummock/store/version.rs

```diff
@@ -235,3 +260,369 @@ impl HummockReadVersion {
         self.staging.sst.clear();
     }
 }
+
+pub struct HummockReadSnapshot(
```

Comment:
Suggest adding some doc to describe the relationship between `HummockReadSnapshot` and `HummockReadVersion` since their fields seem similar.

Reply:
Thanks, remove the concept `HummockReadSnapshot `

Reply:
Also update the description of PR?🥰